go/engine/wot_test.go

```diff
@@ -10,6 +10,7 @@ import (
 )
 
 func TestWebOfTrustVouch(t *testing.T) {
+	t.Skip()
```

Comment:
To be sure not to accidentally leave tests skipped forever I like to do something like this:
```
const temporarySkip = "Temporarily skipped. PICNIC-xxx re-enable after vouchtext change"
t.Skip(temporarySkip)
```
That way removing `temporarySkip` causes compile errors until all's unskipped. Do whatever, just wanted to share.